src/index.js

```diff
@@ -204,20 +246,5 @@ function joinClasses(...classes) {
   return classes.filter(Boolean).join(' ')
 }
 
-// some manual magic umd here because Rollup isn't capable of
```

Comment:
Thanks for removing this. But I think we might need it (as mentioned earlier). 😅